src/java.base/share/classes/sun/security/util/Debug.java

```diff
@@ -87,10 +87,12 @@ public static void Help()
         System.err.println("pkcs11keystore");
         System.err.println("              PKCS11 KeyStore debugging");
         System.err.println("pkcs12        PKCS12 KeyStore debugging");
+        System.err.println("properties    Prints which java.security Security Properties file that the JDK uses");
```

Comment:
This prints the initial values of all security properties (whether overridden or not) and other miscellaneous useful info when parsing the java.security file. I think we can just generalize this to "Security property and configuration file debugging". The other help options are fairly general.

Reply:
Done, thanks Sean